cmd/estat.py

```diff
@@ -439,8 +439,12 @@ class Args:
                                 "avg latency(us)")
         helper1.add_aggregation("lats", BCCHelper.STDDEV_AGGREGATION,
                                 "stddev(us)")
-        helper1.add_aggregation("data", BCCHelper.SUM_AGGREGATION,
-                                "throughput(k/s)")
+        if args.normalize:
```

Comment:
As we discussed offline, it would probably be simplest to remove the `normalize` option entirely and just make all output normalized as humans would expect. This also simplifies the code.